regexes/device/televisions.yml

```diff
@@ -213,7 +213,7 @@ Sony:
   regex: 'Sony'
   device: 'tv'
   models:
-    - regex: '(KDL[0-9]{2}[A-Z]{1,2}[0-9]{3})'
+    - regex: '(KD[L\-]{0,2}[0-9]{2}[A-Z]{1,2}[0-9]{3,4}[A-Z]{0,1})'
```

Comment:
> `KD[L\-]{0,2}`

This might match incorrect stuff, as it also matches `KDLL`, `KD-L` or `KD--` Guess it should match `KDL`, `KDL-`, `KD` and `KD-`, so try `KDL?-?`

And instead of `{0,1}` your could simply use a `?`